redis/src/main/scala/zio/redis/options/Connection.scala

```diff
@@ -82,11 +148,269 @@ trait Connection {
     events: ClientEvents = ClientEvents(),
     lastCommand: Option[String] = None,
     argvMemory: Option[Long] = None,
+    multiMemory: Option[Long] = None,
     totalMemory: Option[Long] = None,
+    replyBufferSize: Option[Long] = None,
+    replyBufferPeakPosition: Option[Long] = None,
     redirectionClientId: Option[Long] = None,
+    respProtocolVersion: Option[String] = None,
     user: Option[String] = None
   )
 
+  object ClientInfoCodec extends zio.schema.codec.Codec {
```

Comment:
Not sure if we should place this next to command options.

Reply:
where should we put it?

Reply:
It's just that I find it a bit confusing here, but since I have no better proposal, let's leave it.